wolfcrypt/src/include.am

```diff
@@ -1,13 +1,14 @@
 # vim:ft=automake
 # All paths should be given relative to the root
 
-BUILT_SOURCES+= wolfcrypt/src/port/cavium/cavium_nitrox.c
-BUILT_SOURCES+= wolfcrypt/src/port/intel/quickassist.c
-BUILT_SOURCES+= wolfcrypt/src/port/intel/quickassist_mem.c
+QUICK_ASSIST_FILES =					\
```

Comment:
Can you rename this to `ASYNC_FILES`? The quick assist term is specific to Intel. I ran this on actual hardware to confirm it still works.

Reply:
Done and rebased.